packages/edit-site/src/components/sidebar-navigation-screen-global-styles/index.js

```diff
@@ -94,6 +94,7 @@ function SidebarNavigationScreenGlobalStylesContent() {
 				) }
 				{ typographyVariations?.length && (
 					<TypographyVariations
+						addGlobalStyles
```

Comment:
I am passing this prop as true here, because I am thinking we should keep the color combination in the sidebar navigation global screen, Let me know if we need to remove the color from here as well, I will update the PR.

Thank You.

Reply:
I think we should remove it here.

Reply:
Sure, I will work on this PR and update it by removing the color from here as well. 

So after update, both the global screen and side panel would not have the color effect in the typography. Am I correct?

Reply:
Yeah, I had a different idea for how to do it which I tried here: https://github.com/WordPress/gutenberg/pull/61898

I'm not sure which option is better, what do you think?

Reply:
Yeah, looks like the good option. Just add the styles to iframe whatever we want using this prop.